cmd/secret/add.go

```diff
@@ -84,13 +84,8 @@ var AddCmd = cli.Command{
 		},
 		&cli.StringSliceFlag{
 			Name:    "event",
-			Usage:   "Secret limited to these events",
+			Usage:   "Secret limited to these events (by default push, tag, and deploy are enabled)",
```

Comment:
not super happy about this as we will have to maintain, although it shouldn't update _that_ often.